include/dungeon_global_data.h

```diff
@@ -0,0 +1,93 @@
+#ifndef GUARD_DUNGEON_DATA_H
+#define GUARD_DUNGEON_DATA_H
+
+#define DUNGEON_SIZE_X 55
```

Comment:
Are they always 55 x 31 or is this supposed to be like the max?

Reply:
This is the max. There are places with bounds checks for this.

Reply:
Probably `DUNGEON_MAX_SIZE_X`  / `DUNGEON_MAX_SIZE_Y` might get that point across then

Reply:
Renamed.